src/app/dashboard/@navbar/page.tsx

```diff
@@ -0,0 +1,91 @@
+import Box from '@mui/material/Box';
+import IconButton from '@mui/material/IconButton';
+import MenuIcon from '@mui/icons-material/Menu';
+import SvgIcon from '@mui/material/SvgIcon';
+import styles from './styles.module.css';
+import Typography from '@mui/material/Typography';
+
+const BrandIcon = () => (
+  <SvgIcon>
+    <svg
+      width="24"
+      height="24"
+      viewBox="0 0 33 33"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <rect
+        x="17.678"
+        y="1.57644"
+        width="17.9566"
+        height="22.9421"
+        rx="2.86322"
+        transform="rotate(30 17.678 1.57644)"
+        fill="#FFF176"
+        stroke="black"
+        stroke-width="1.43161"
```

Comment:
This attribute is causing warnings in the chrome console, replace it with the camelCase version:
```suggestion
        strokeWidth="1.43161"
```

Warning from chrome:
```
app-index.js:34 Warning: Invalid DOM property `stroke-width`. Did you mean `strokeWidth`?
    at rect
    at svg
```